### PR TITLE
[collect_tests] Add vulture_whitelist files to IGNORED_FILE_TYPES

### PR DESCRIPTION
**Pull request by mmhw**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
collect_tests will be skipped .vulture_whitelist.py files.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by mmhw** [comment]
Will merge only after an `demisto-sdk` version with [this PR](https://github.com/demisto/demisto-sdk/pull/3633) is released.

File: Tests/scripts/infrastructure_tests/test_collect_tests.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
+import logging
```
Comment (dorschw) [inline Tests/scripts/infrastructure_tests/test_collect_tests.py:1]: let's keep this PR minimal, undo this change. We'll merge with `print`s.

File: Tests/scripts/infrastructure_tests/test_collect_tests.py [hunk 2]
```diff
@@ -632,7 +634,7 @@ def test_number_of_file_types():
 
         - Removed type:    Decrease the number here.
     """
-    assert len(FileType) == 76
+    assert len(FileType) == 77
```
Comment (dorschw) [inline Tests/scripts/infrastructure_tests/test_collect_tests.py:637]: have you added it to `IGNORED_FILE_TYPES`?

Comment (mmhw) [reply Tests/scripts/infrastructure_tests/test_collect_tests.py:637]: https://github.com/demisto/content/pull/29667/files#diff-2a6c9234e44f974b6213005e4fcff734db5dd00c16d48b377859d606b215ea81R117

**Comment by kobymeir** [comment]
merged by https://github.com/demisto/content/pull/29721
